ts/packages/ts/src/commands/TestCommand.ts

```diff
@@ -0,0 +1,32 @@
+import { Command } from '../Command';
+import { ConfigStore } from '../ConfigStore';
+
+export type TestCommandParameters = {
+  store: ConfigStore;
+  clean?: boolean;
+};
+
+export class TestCommand extends Command<boolean> {
+  constructor(public parameters: TestCommandParameters) {
+    super(parameters);
+  }
+
+  async run(): Promise<boolean> {
+    const testConfig = {
+      set: '',
+      key: 'CONFIGU_TEST',
+      value: Date.now().toString(),
+    };
+    try {
```

Comment:
see previous comment regarding error, dump try catch and let the store.set() to throw the "real" error.

Reply:
done